modules/codegen/src/smithy4s/codegen/internals/ModelLoader.scala

```diff
@@ -51,7 +51,7 @@ private[codegen] object ModelLoader {
     val modelsInJars = deps.flatMap { file =>
       Using.resource(
         // Note: On JDK13+, the second parameter is redundant.
-        FileSystems.newFileSystem(file.toPath(), null)
+        FileSystems.newFileSystem(file.toPath(), null: ClassLoader)
```

Comment:
**thought:** this was causing trouble for my sbt (@ghostbuster91 also reported seeing this):

```
[error] [codegen2_12] /Users/kubukoz/projects/smithy4s/modules/codegen/src/smithy4s/codegen/internals/ModelLoader.scala:54:21: ambiguous reference to overloaded definition,
[error] [codegen2_12] both method newFileSystem in class FileSystems of type (x$1: java.nio.file.Path, x$2: java.util.Map[String, _])java.nio.file.FileSystem
[error] [codegen2_12] and  method newFileSystem in class FileSystems of type (x$1: java.nio.file.Path, x$2: ClassLoader)java.nio.file.FileSystem
[error] [codegen2_12] match argument types (java.nio.file.Path,Null)
[error] [codegen2_12]         FileSystems.newFileSystem(file.toPath(), null)
[error] [codegen2_12]                     ^
[error] [codegen2_12] one error found
[error] [codegen2_12] (codegen2_12 / Compile / compileIncremental) Compilation failed
```

I'm not sure why it's failing now, but I'm sure it was fine a couple months ago. Anyway, doesn't seem important enough to investigate.